internal/transport/controlbuf.go

```diff
@@ -650,16 +650,19 @@ func (l *loopyWriter) headerHandler(h *headerFrame) error {
 		itl:   &itemList{},
 		wq:    h.wq,
 	}
-	str.itl.enqueue(h)
-	return l.originateStream(str)
+	return l.originateStream(str, h)
 }
 
-func (l *loopyWriter) originateStream(str *outStream) error {
-	hdr := str.itl.dequeue().(*headerFrame)
+// originateStreamWithHeaderFrame calls the initStream function on the headerFrame and
+// called writeHeader. If write succeeds the streamID is added to l.estdStreams
```

Comment:
called->calls?

And end the last sentence with a period, please.

Reply:
I nixed the comment itself. Seems like i really just wrote exactly what the below 10 lines do